website/docs/r/db_instance.html.markdown

```diff
@@ -133,7 +133,7 @@ if omitted, Terraform will assign a random, unique identifier. Required if `rest
 identifier beginning with the specified prefix. Conflicts with `identifier`.
 * `instance_class` - (Required) The instance type of the RDS instance.
 * `iops` - (Optional) The amount of provisioned IOPS. Setting this implies a
-storage_type of "io1".
+storage_type of "io1" or "gp3".
```

Comment:
This will need to be reworded. The sentence

> Setting this implies a storage_type of "io1".

means that if `iops` is set and `storage_type` is not, then `storage_type` will be automatically set to `io1`. This is still the case, even when support for `gp3` is added.

Instead, we should add a sentence stating something like:

> Can only be set when `storage_type` is `io1` or `gp3`